src/platforms/android/common/enriching-events/attachments.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: "Attachments Preview"
+sidebar_order: 106
+
+description: "Learn more about attachments, which provide the ability to store additional files to an event."
+---
+
+__Attention: This is a preview API, available since 4.0.0-alpha.2. This API is not in stable state yet. It may be renamed, changed or even removed in a future version.__
+
+Attachments provide the ability to to store additional files to an event. Be aware that attachments don't work yet with crashes.
+
+
+## Uploading Attachments
+
+You can either pass bytes or use a pathname to a file to create an attachment.
+
+```java
```

Comment:
can we use the Tabs feature instead of adding a code snippet below?

Reply:
What exactly is the Tabs feature?

Reply:
oh looks like it works by default?!

<img width="203" alt="Screenshot 2020-12-17 at 10 50 34" src="https://user-images.githubusercontent.com/5731772/102471610-b2237c80-4055-11eb-908d-b2cb50aa3ec4.png">

if you look at https://raw.githubusercontent.com/getsentry/sentry-docs/master/src/platforms/java/usage.mdx

we needed to add this before: `{tabTitle:LanguageX}` so it'd group otherwise not.

Reply:
I think `{tabTitle:LanguageX}` only makes sense when the language specification for the code snippet deviates from the actual language name, for example `objc {tabTitle:Objective-C}`.

Reply:
they probably fixed this, as I stated before, without the `tabTitle`, they were not grouping at all.

Reply:
Ah okay didn't know that.